conf.yaml

```diff
@@ -1117,8 +1117,8 @@ contents:
                   - title:      APM .NET Agent
                     prefix:     dotnet
                     current:    1.8
-                    branches:   [ master, 1.x, 1.8 ]
-                    live:       [ master, 1.x, 1.8 ]
+                    branches:   [ master, 1.8 ]
+                    live:       [ master, 1.8 ]
```

Comment:
If `live` is all of the `branches`, we can just leave it out. But it might be good to leave it here because I don't think that will always be true.

Reply:
+1 for leaving it as it's only a matter of time until they're different.